MiniApp/Classes/Logger.swift

```diff
@@ -1,37 +0,0 @@
-import os
-
-extension OSLog {
-    static let sdk = OSLog(category: "Mini App SDK")
-
-    private convenience init(category: String, bundle: Bundle = Bundle(for: Logger.self)) {
-        let identifier = bundle.infoDictionary?["CFBundleIdentifier"] as? String
-        self.init(subsystem: (identifier ?? "").appending(".logs"), category: category)
-    }
-}
-
-internal class Logger {
```

Comment:
It seems like this class is very helpful for printing debug messages, such as in the above case. Even if it's not being used much right now, maybe it will still be useful?

Reply:
Yeah, I don't understand why a configurable utility class like Logger has been deleted.

Reply:
I also don't see any good reason for removing this from the SDK code.

Reply:
In fact, should also get branded to be `MiniAppLogger` or alike.

Reply:
Noted. I think we should start using this class more in the SDK. Renaming the class with a new name